src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -17,6 +18,11 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Private
 
+(def OIDMap
+  "Mapping of string OIDs to shortname keywords. Used to update an incoming
+  request with a shortname -> value extensions map."
+  {String schema/Keyword})
```

Comment:
It's not consistent : / I see both in our code. I can switch this to schema/Str so it's at least consistently using the schema ns in this schema...